frontend/components/transaction/ExecutionBlock.vue

```diff
@@ -64,23 +45,17 @@
 import Vue from 'vue';
 import BigNumber from 'bignumber.js';
 import { Alert } from 'ant-design-vue';
-import TextBlock from '~/components/common/TextBlock.vue';
 import BaseButton from '~/components/common/BaseButton.vue';
-import FormatAddress from '~/components/utils/FormatAddress.vue';
-import FormatCurrency from '~/components/utils/FormatCurrency.vue';
-import Explain from '~/components/utils/Explain.vue';
 import ExecuteWithOtherWalletBlock from '~/components/transaction/ExecuteWithOtherWalletBlock.vue';
+import TransactionMessage from '~/components/transaction/TransactionMessage.vue';
```

Comment:
Can you please remove `transactionURL` from the `ExecutionBlock.vue` component? it's unrelated to your PR, but unused